proc-macros/src/helpers.rs

```diff
@@ -149,3 +149,15 @@ fn visit_trait(item_trait: &syn::ItemTrait, sub_tys: &[syn::Type]) -> FindAllPar
 	visitor.visit_item_trait(item_trait);
 	visitor
 }
+
+/// Checks whether provided type is an `Option<...>`.
+pub(crate) fn is_option(ty: &syn::Type) -> bool {
+	if let syn::Type::Path(path) = ty {
+		// TODO: Probably not the best way to check whether type is an `Option`.
```

Comment:
I guess a slightly better version would be to check only the last segment? (I'd have to read the `syn` manual to remind myself what these things are!)

Reply:
sure but nothing is introduced in this PR, I can add an issue for this ^^